src/commands/run.rs

```diff
@@ -2,13 +2,11 @@
 use crate::server::serve;
 use clap::Parser;
 use nix::unistd;
-use nix::unistd::{dup2, fork, ForkResult};
-use std::fs::File;
-use std::fs::OpenOptions;
+use nix::unistd::{fork, ForkResult};
+
 use std::io::Error;
-use std::io::Write;
+
```

Comment:
Any reason for the empty lines here?

Reply:
No just me manually deleting imports and not the whole lines
For some reason my rust integration is fine with adding missing imports but does not remove them automatically.

Reply:
I will fix this up tomorrow.